adjustcopyright.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python
+import sys
+import os
+import fileinput
+import re
+from datetime import datetime, timedelta
+
+# argument handling
+if len(sys.argv) != 3:
+    print("you must set two arguments. [this file] [root dir] [version number]")
+    exit()
+
+rootdir = sys.argv[1]
+version = sys.argv[2]
+
+# filters
+fileExtensionFilter = ('.c', '.h', '.cybol', '.sh', '.txt', '.xsd', '.css', '.dtd', '.html', '.py')
+fileFilter = ('AUTHORS', 'ChangeLog', 'COPYING', 'INSTALL', 'NEWS', 'README')
+folderExclude = ('include', 'tools')
```

Comment:
please remove the 'include' from the folderExcludes. From now on there are h files in this folder that need copyright adjustments as well

Reply:
Since the renamed (all small letter) file "adjustcopyright.py" got deleted,
I continued in file "adjustCopyright.py".
I've removed the "include" directory from "folderExclude",
but also added the "bin" and "lib" directories.